config/locales/en.yml

```diff
@@ -18,3 +19,6 @@ en:
     assignee set, tick the box below to assign the issue to the user who just
     created the issue, if possible. If the box is left unticked, there will be
     no assignee, which is the default behaviour in Redmine.
+  default_assign:
+    override_category_assignee_info: "Override Category Assignee"
+    override_category_assignee: "Override Category Assignee"
```

Comment:
You might need to add a sentence in English to the end of whatever they have for the `interactive_assignment_explanation` text. Not ideal, but at least then it'll get picked up by the translators.